integrations/stellar.go

```diff
@@ -0,0 +1,131 @@
+package integrations
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/stellar/go/txnbuild"
+	"github.com/stellar/starbridge/model"
+)
+
+// TODO need to set the contract account, source account
+var sourceAccount = "GDXLLPH23EHIFOQLO46X2WQQPBBRJ6YPV7JOXEWS7V3AU74Z4EY7PGCS" // var sourceSecretKey = "SAJOMEU6AAHWIUSF43Z7BGFEEB4VUCZVTG56U4DU6RR3UGAZRFSHEYEQ"
+
+var baseFee int64 = 100
+
+func getStellarAsset(assetInfo *model.AssetInfo) txnbuild.Asset {
```

Comment:
since we use assetInfo defined in chain.go it is very easy to convert between currencies from different chains!